src/ingredient.js

```diff
@@ -1,8 +1,8 @@
 class Ingredient {
-  constructor(ingredient, recipeIngredientData) {
-    this.id = ingredient.id,
+  constructor(dataIngredient, recipeIngredientData) {
+    this.id = dataIngredient.id,
     this.name = recipeIngredientData.name,
```

Comment:
Just a note, this will have to be changed to ```this.name = dataIngredient.name``` because our API server data does not have a name property with the recipeIngredientData. It might help to rename these parameters too but I appreciate that they are weird ingredients to distinguish.